### PR TITLE
Alter sidebar

### LINKED ISSUE #228: Need to show the sidebar in all the pages

### PR DESCRIPTION
**Pull request by Micha546**
# Description
Moving the side bar to the base_template

Now we can see the side bar in all the pages.
Added an empty sidebar tag in apartment_register and seeker_register
So there will be no sidebar there.
Deleted home.html as it is no longer in use

## Screenshots
image
image

### Issues closed by this PR
Fix #228

### PR Dependencies
Depended on #226

### CODE REVIEW

**Review by DanielMy777 — APPROVED** [review]
Considering you checked and this works
LGTM

**Comment by nadavsu** [comment]
In the images, the side bar has no name.
It says "Welcome, !". It should say the user's name though.

I'm assuming it's because these stuff aren't related to the PR, but it should be fixed in another PR later.

**Comment by nadavsu** [comment]
Also, you need to rebase here I think.
Other than that, it looks good.

File: main/templates/main/base_template.html [hunk 1]
```diff
@@ -87,6 +87,17 @@
   <main class="row main-body">
     <div class="col-md-2">
       {% block sidebar %}
+        <div class="card shadow p-3 mb-5 bg-white rounded">
+          <img src="{{ request.user.image_url }}" alt="Profile picture" id="profile_picture">
+            <div class="card-body">
+              <h5 class="card-title username">Welcome, {{seeker.first_name}}!</h5>
```
Comment (DanielMy777) [inline main/templates/main/base_template.html]: Switch this to `{{user.first_name}}`

**Review by DanielMy777 — CHANGES_REQUESTED** [review]

**Review by DanielMy777 — APPROVED** [review]
LGTM

**Review by nadavsu — COMMENTED** [review]
Nice.

**Pull request merged**